Jenkinsfile.github

```diff
@@ -13,37 +39,48 @@ pipeline {
     environment {
         NAME = "pit-nexus"
         DESCRIPTION = "Nexus scripts and resources for inclusion in the Cray Pre-Install Toolkit"
-        IS_STABLE = getBuildIsStable()
-        BUILD_METADATA = getRpmRevision(isStable: env.IS_STABLE)
+        IS_STABLE = "${isStable}"
+        BUILD_METADATA = getRpmRevision(isStable: isStable)
+        VERSION = getDockerBuildVersion(isStable: isStable)
+        GIT_REPO_NAME = getRepoName()
+        DOCKER_ARGS = getDockerBuildArgs(name: getRepoName(), description: env.DESCRIPTION)
     }
 
     stages {
-        stage("Build") {
+        stage("Prepare") {
+             steps {
+             runLibraryScript("addRpmMetaData.sh", env.SPEC_FILE)
+             sh "make prepare"
+           }
+        }
+
+        stage("Build: RPM") {
             agent {
                 docker {
-                    image 'artifactory.algol60.net/csm-docker/stable/sle15sp3_build_environment'
+                    label 'docker'
+                    image "artifactory.algol60.net/csm-docker/stable/sle15sp3_build_environment:latest"
                     reuseNode true
                     registryUrl 'https://artifactory.algol60.net/'
                     registryCredentialsId 'artifactory-algol60'
                     // Run as root
-                    args "-u root -v /var/run/docker.sock:/var/run/docker.sock -v /usr/bin/docker:/usr/bin/docker --group-add 999"
```

Comment:
I'm curious why this is not needed anymore.

Reply:
Fixed